devtools/maven/src/main/java/io/quarkus/maven/AbstractImageMojo.java

```diff
@@ -63,7 +63,8 @@ protected Optional<ArtifactDependency> getContainerImageExtension(Builder builde
     protected Optional<ArtifactDependency> getExtension(String artifactId) {
         return mavenProject().getDependencyManagement().getDependencies().stream()
                 .filter(d -> "io.quarkus".equals(d.getGroupId()) && artifactId.equals(d.getArtifactId()))
-                .map(d -> new ArtifactDependency(d.getGroupId(), d.getArtifactId(), null, ArtifactCoords.TYPE_JAR,
+                .map(d -> new ArtifactDependency(d.getGroupId(), d.getArtifactId(), null,
+                        io.quarkus.maven.dependency.ArtifactCoords.TYPE_JAR,
```

Comment:
If you had to use a full package and the previous package is not gone, that would mean we still use the type somewhere in the class?

Reply:
It's in the same package as this class